src/App.vue

```diff
@@ -533,15 +534,15 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // now that we have user info and org info, populate the completing party
     // NB: these are all empty for staff
     this.setCompletingParty({
-      firstName: this.isRoleStaff ? '' : this.getUserFirstName,
-      lastName: this.isRoleStaff ? '' : this.getUserLastName,
+      firstName: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getUserFirstName,
+      lastName: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getUserLastName,
       mailingAddress: {
-        addressCity: this.getOrgInfo?.mailingAddress.city,
-        addressCountry: this.getOrgInfo?.mailingAddress.country,
-        addressRegion: this.getOrgInfo?.mailingAddress.region,
-        postalCode: this.getOrgInfo?.mailingAddress.postalCode,
-        streetAddress: this.getOrgInfo?.mailingAddress.street,
-        streetAddressAdditional: this.getOrgInfo?.mailingAddress.streetAdditional
+        addressCity: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.city,
+        addressCountry: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.country,
+        addressRegion: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.region,
+        postalCode: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.postalCode,
+        streetAddress: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.street,
+        streetAddressAdditional: (this.isRoleStaff || this.isSbcStaff) ? '' : this.getOrgInfo?.mailingAddress.streetAdditional
```

Comment:
Instead of computing this 8 times, I recommend adding a local variable and computing it just once. This will also make the code look cleaner.

A more generic solution would be to add a store getter for "is role staff or sbc staff" and use that getter here.

Reply:
Sorry, I missed the local variable. Looks fine 👍